src/opnsense/mvc/app/models/OPNsense/IPsec/Menu/Menu.xml

```diff
@@ -0,0 +1,7 @@
+<menu>
+    <VPN>
+        <IPsec>
+            <PublicKeys order="31" VisibleName="RSA Key Pairs" url="/ui/ipsec/key-pairs" />
```

Comment:
would you mind moving all IPsec menu items to this location? the IPsec ACL can be moved in a similar way.

Reply:
Done, I have moved all the legacy ACL and menu entries into the MVC code in a separate commit, as other parts than described in this PR have been touched. Additionally, this revealed a 'bug' in the current code, where the ACL `page-status-systemlogs-ppp` has been mislabeled as `IPsec VPN`, which I also fixed in this commit.